api-catalog-services/src/test/java/org/zowe/apiml/apicatalog/functional/AttlsConfigTest.java

```diff
@@ -57,7 +61,7 @@ void requestFailsWithAttlsContextReasonWithHttp() {
                 given()
                     .log().all()
                 .when()
-                    .get(getCatalogUriWithPath("http", "apicatalog/api/v1/containers"))
+                    .get(getCatalogUriWithPath("http", "apicatalog/containers"))
```

Comment:
@achmelo this may be related to the API Catalog issues.

Reply:
are there still any issues?

Reply:
no, it's working fine now :)